app.spec.js

```diff
@@ -11,6 +11,6 @@ jest.mock('./config/routes.config', () => {
 })
 
 test('Server can request first route and receive 200 response', async () => {
-  const response = await request(app).get(routes[0].path)
+  const response = await request(app).get(app.routes.get('en', 'start').path)
```

Comment:
Original intent here was to not break test if someone didn't have a start route - I think you have a test routes config in the other PR so should be fine.